starter-web-netty/README.md

```diff
@@ -33,7 +33,7 @@ $ docker push gcr.io/[PROJECT ID]/micronaut-starter
 You are now ready to deploy your application:
 
 ```
-$ gcloud run deploy --image gcr.io/[PROJECT ID]/micronaut-starter --platform=managed --allow-unauthenticated
+$ gcloud run deploy --image gcr.io/[PROJECT ID]/micronaut-starter --set-env-vars=GITHUB_OAUTH_APP_CLIENT_ID=[CLIENT ID],GITHUB_OAUTH_APP_CLIENT_SECRET=[SECRET ID] --platform=managed --allow-unauthenticated
```

Comment:
We believe `set-env-vars` removes any existing vars as well, think we should change this to `--update-env-vars` which updates but doesn't remove.